bitcoin/src/test_macros.rs

```diff
@@ -1,16 +0,0 @@
-// SPDX-License-Identifier: CC0-1.0
-
-//! Bitcoin serde macros.
-//!
-//! This module provides internal macros used for unit tests.
-
-#[cfg(feature = "serde")]
-macro_rules! serde_round_trip (
-    ($var:expr) => ({
-        use serde_json;
-
-        let encoded = serde_json::to_value(&$var).unwrap();
-        let decoded = serde_json::from_value(encoded).unwrap();
-        assert_eq!($var, decoded);
```

Comment:
Seeing the actual code, I think we should really have `test-serde` feature that depends on `serde_json`, some binary `serde` format and uses `$crate::serde_json` and also checks roundtrip for the binary format (to test the correctness of `is_human_readable` dispatch). The feature will make it more convenient to use (depend on one thing instead of two).